samza-core/src/main/scala/org/apache/samza/diagnostics/DiagnosticsStreamMessage.java

```diff
@@ -60,17 +62,18 @@
   private static final String CONTAINER_THREAD_POOL_SIZE_METRIC_NAME = "containerThreadPoolSize";
   private static final String CONTAINER_MODELS_METRIC_NAME = "containerModels";
   private static final String AUTOSIZING_ENABLED_METRIC_NAME = "autosizingEnabled";
+  private static final String CONFIG = "config";
 
   private final MetricsHeader metricsHeader;
   private final Map<String, Map<String, Object>> metricsMessage;
 
   public DiagnosticsStreamMessage(String jobName, String jobId, String containerName, String executionEnvContainerId,
-      String taskClassVersion, String samzaVersion, String hostname, long timestamp, long resetTimestamp) {
+      String taskClassVersion, String samzaVersion, String hostname, long timestamp, long resetTimestamp, Map<String, String> config) {
 
     // Create the metricHeader
     metricsHeader =
         new MetricsHeader(jobName, jobId, containerName, executionEnvContainerId, DiagnosticsManager.class.getName(),
-            taskClassVersion, samzaVersion, hostname, timestamp, resetTimestamp);
+            taskClassVersion, samzaVersion, hostname, timestamp, resetTimestamp, config);
```

Comment:
See comment above, 
config is not required to be in the header.

Reply:
Removed.